enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -1466,17 +1466,17 @@
             (let [built-in-action-id "dashcard:unknown:built-in-create"
                   custom-action-id   "dashcard:unknown:custom-create"
                   scope              {:dashcard-id (:id dashcard)}]
-
               (testing "built-in"
-                (is (=? {:parameters
-                         ;; params are reordered by editable
-                         ;; column listing (int first)
-                         [{:id "int" :readonly false}
-                          {:id "text" :readonly true :value "a very important string"}
-                          ;; date is hidden from the editable
-                          #_{:id "date"}
-                          ;; timestamp is hidden in the row action
-                          #_{:id "timestamp"}]}
+                (is (=? {:title      (format "%s: Create" (t2/select-one-fn :name :model/Table @test-table))
+                         :parameters [;; params are reordered by editable
+                                      ;; column listing (int first)
+
+                                      {:id "int" :readonly false}
+                                      {:id "text" :readonly false #_:value #_"a very important string"}
+                                      ;; date is hidden from the editable
+                                      #_{:id "date"}
```

Comment:
need to call out some changes I made to the test here @crisptrutski .

this is a built-in action, so it's not configurable => 
- `text` column won't have value and readonly status
- timeestamp is not hidden, it's only hidden for the custom one.